ksqldb-engine/src/main/java/io/confluent/ksql/analyzer/AggregateAnalyzer.java

```diff
@@ -204,9 +226,18 @@ private void throwOnWindowBoundColumnIfWindowedAggregate(
 
     private void captureNonAggregateSelectNotPartOfGroupBy(
         final Expression expression,
-        final Set<ColumnReferenceExp> nonAggParams
+        final Set<Expression> nonAggParams
     ) {
-      final boolean matchesGroupBy = groupBy.contains(expression);
+
+      boolean matchesGroupBy = false;
+      // If the non-Agg expression is a function, then all its arguments must be part of the
+      // grouping columns. Note, they may be nested inside other functions.
+      /*if (expression instanceof FunctionCall) {
+        matchesGroupBy = functionContainsOnlyGroupingColumns(expression, true);
+      } else {
+        matchesGroupBy = groupBy.contains(expression);
+      }*/
```

Comment:
is there a reason this code is still commented out?

Reply:
Sorry, forgot to remove

Reply:
+1